jobs/gtfs-rt-parser/gtfs_rt_parser.py

```diff
@@ -193,73 +186,48 @@ def try_handle_one_feed(
         return e
 
 
-class Prefix(str, Enum):
-    al = "al"
-    tu = "tu"
-    rt = "rt"
-
-
-class RTFileType(str, Enum):
-    service_alerts = "service_alerts"
-    trip_updates = "trip_updates"
-    vehicle_positions = "vehicle_positions"
-
-
-yesterday = (date.today() - timedelta(days=1)).isoformat()
-
-
 def main(
-    filename_prefix: Prefix,
-    rt_file_substring: RTFileType,
-    iso_date: str = yesterday,
-    src_path=f"{get_bucket()}/rt/",
-    dst_path=f"{get_bucket()}/rt-processed/",
+    file_type: RTFileType,
+    glob: str = f"{get_bucket()}/rt/{yesterday}T12:*",
+    dst_bucket=get_bucket(),
     limit: int = 0,
     progress: bool = False,
```

Comment:
**question:** Can we add a comment or something somewhere saying that `progress` is just for local testing to show progress through loops? Or maybe rename it to like `show_progress`? I guess maybe this is standard and most people would know it but I was just kind of tripped up seeing the different `if` blocks and not understanding the purpose

Reply:
Yep will do!